app/src/main/java/com/globant/marvelmvvm/view/fragment/AllCharactersFragment.kt

```diff
@@ -66,16 +71,25 @@ class AllCharactersFragment : Fragment() {
     }
 
     private fun showAllCharacters(data: List<Character>?) {
-        setLoaderState(View.GONE)
-        data.let {
-            fragment_all_characters_character_id.text =
-                "${getString(R.string.string_first_character_id)} ${it?.get(ZERO)?.id}"
-            fragment_all_characters_count.text =
-                "${getString(R.string.string_number_of_characters)} ${it?.size}"
+        data?.let {
+            setLoaderState(View.GONE)
+            allCharactersAdapter.submitList(it)
+            fragment_all_characters_recycler_view.apply {
+                layoutManager = LinearLayoutManager(this.context)
+                adapter = allCharactersAdapter
+                visibility = View.VISIBLE
+            }
         }
     }
 
     private fun showError(){
         Toast.makeText(this.context, getString(R.string.string_request_error), Toast.LENGTH_SHORT).show()
     }
+
+    private fun replaceFragment(characterId: String){
+        val navController = this.findNavController()
```

Comment:
you can use navController just using findNavController().navigate(...) or if you are going to use it in another function pls declare it at the beginning of the class

Reply:
Yes, you are right Nico, thanks!